### PR TITLE
Set refs

### PR DESCRIPTION
**Pull request by satr**

### CODE REVIEW

File: pkg/models/env/env.go [hunk 1]
```diff
@@ -94,6 +101,25 @@ func (e *env) GetRadixDeployToEnvironment() string {
...
 
+// GetDNSConfig The list of DNS aliases, reserved for Radix platform Radix application and services
+func (e *env) GetDNSConfig() *dnsalias.DNSConfig {
+	return e.dnsConfig
+}
+
+func validateReservedDNSAliases(dnsConfig *dnsalias.DNSConfig) {
+	var errs []error
+	if len(dnsConfig.ReservedAppDNSAliases) == 0 {
+		errs = append(errs, fmt.Errorf("missing DNS aliases, reserved for Radix platform Radix application"))
```
Comment (Richard87) [inline pkg/models/env/env.go:112]: Can we make these errors static?

**Review by Richard87 — CHANGES_REQUESTED** [review]
Should/could errors be static?

File: pkg/models/env/env.go (hunk 1 above)
Comment (satr) [reply pkg/models/env/env.go]: Fixed

**Review by Richard87 — APPROVED** [review]
lgtm

**Pull request merged**